### ISSUE TITLE
variable length sequence

### ISSUE THREAD

**Issue by rubick1896** [martenlienen/torchode#21]
Is it possible to integrate a batch with not only different time spans but also a different number of observations? In the standard neural network approach, we pad the sequence with a special token and mask out those positions. Is there an equivalent way in ODE?

**Comment by martenlienen** [comment]
The equivalent for torchode would be to repeat the last evaluation time, i.e.

```python
t = torch.tensor([[0.1, 0.2, 0.3], [0.0, 0.5, 0.5]])
```

Then it would evaluate the solution at 0.5 twice. I think the implementation would be extensible to support a mask so that the solution would only be evaluated once at 0.5 for the second sample and would be open for a pull request that implements that.
